doc/man/scons.xml

```diff
@@ -163,128 +163,158 @@ options for details on default paths and
 import os
 env = Environment(ENV=os.environ)
-</literallayout>
+</programlisting>
 
 <para>This comes at the expense of making your build
 dependent on the user's environment being set correctly,
 but it may be more convenient for many configurations.
 It should not cause problems if done in a build setup which tightly
 controls how the environment is set up before invoking
-<command>scons</command>, as in many continuous
+&scons;, as in many continuous
 integration setups.
 </para>
 
-<para><command>scons</command>
+<para>&scons;
 can scan known input files automatically for dependency
 information (for example, <literal>#include</literal>
 preprocessor directives in C or C++ files)
 and will rebuild dependent files appropriately
 whenever any "included" input file changes.
-<command>scons</command>
+&scons;
 supports the
 ability to define new scanners for unknown input file types.</para>
 
-<para><command>scons</command>
+<para>&scons;
 knows how to fetch files automatically from
 SCCS or RCS subdirectories
 using SCCS, RCS or BitKeeper.</para>
 
-<para><command>scons</command>
+<para>&scons;
 is normally executed in a top-level directory containing an
 &SConstruct; file.
-When <command>scons</command> is invoked,
+When &scons; is invoked,
 the command line (including the contents
 of the &SCONSFLAGS; environment variable,
 if set) is processed.
-The options are consumed (see <xref linkend="options"></xref>;
-and also note that SConscript files may define additional options
-for the project using the &AddOption; function).
-Any variable argument assignments (see below) are also consumed.
-Any remaining arguments are taken as the targets to build.
-Targets on the command line may be files, directories,
+Command-line options (see <xref linkend="options"></xref>;) are consumed.
+Any variable argument assignments are collected, and
+remaining arguments are taken as the targets to build.</para>
+
+<para>Values of variables to be passed to the SConscript files
+may be specified on the command line:</para>
+
+<screen>
+<userinput>scons debug=1</userinput>
+</screen>
+
+<para>These variables are made available
+through the &ARGUMENTS; dictionary,
+and can be used in the SConscript files to modify
+the build in any way:</para>
+
+<programlisting language="python">
+if ARGUMENTS.get('debug', 0):
+    env = Environment(CCFLAGS = '-g')
+else:
+    env = Environment()
+</programlisting>
+
+<para>The command-line variable arguments are also available
+in the &ARGLIST; list,
+indexed by their order on the command line.
+This allows you to process them in order rather than by name,
+if necessary. Each &ARGLIST; entry is a tuple containing
+(<replaceable>argname</replaceable>, <replaceable>argvalue</replaceable>).
+</para>
+
+<para>Targets on the command line may be files, directories,
 or phony targets defined using the &Alias; function.
-The command line targets are available in the
+The command line targets are made available in the
```

Comment:
not sure you need "made"

Reply:
okay, I'll wait to see if there are other comments, and I can pull that word back out.